### ISSUE TITLE
label vs major

### ISSUE THREAD

**Issue by bhomass** [yueyu1030/COSINE#25]
what is the difference between label and major in the training data? Not mentioned in the paper of README

**Comment by bhomass** [comment]
I believe label is the true label, major is what came out of the weak learning.

**Comment by yueyu1030** [comment]
Yes, your understanding is correct! `Major` is the aggregated labels over different labeling functions via majority voting.

**Comment by bhomass** [comment]
sorry, I need re-open this issue. isn't whole idea of COSINE, so you would not require manual annotation, and the so called init step uses weak learning (says so explicitly in the paper). So why do you need the ground truth label in the training data?
